src/components/sections/LatticeView.tsx

```diff
@@ -50,92 +65,110 @@ const LatticeView = ({ compilationResult }: LatticeViewProps): JSX.Element => {
     const handleChange = () => {
         setCompilationText(!showCompilationText)
     }
-
+    console.log(circuit_after_litinski)
     return (
-        <div id="lattice-view-output">
-            <h2 ref={latticeSection} className="scroll-margin">
-                Lattice Viewer
-            </h2>
-            <hr />
+        <>
+            <VStack spacing={4} align="stretch">
+                <Heading as="h1" size="2xl" mt={2} mb={2} textAlign={"center"}>
+                    Lattice Viewer
+                </Heading>
+                <Center>
+                    <Box h="10px" bg="blue.200" rounded="lg" width="75%" />
+                </Center>
+            </VStack>
 
-            <div className="form-check form-switch p-1">
-                <div className="form-check form-switch">
-                    <input
-                        className="form-check-input lg-checkbox"
-                        type="checkbox"
-                        role="switch"
-                        id="flexSwitchCheckDefault"
-                        onChange={handleChange}
-                        defaultChecked={true}
-                    />
-                    <label className="form-check-label" htmlFor="flexSwitchCheckDefault">
-                        View Compilation
-                    </label>
-                </div>
-            </div>
+            <Flex gap={3} justifyContent={"center"} py={4} flexWrap={"wrap"}>
+                <FormLabel htmlFor="compilation-text" fontSize="xl" mb="0">
+                    View Compilation
+                </FormLabel>
+                <Switch
+                    id="compilation-text"
+                    size="lg"
+                    onChange={handleChange}
+                    defaultChecked={true}
+                />
+            </Flex>
 
-            <div className="p-1 vertical-center">
-                <div>
-                    {showCompilationText ? (
-                        <div
-                            id="compilation-text"
-                            className="mb-3"
-                            css={css`
-                                margin-left: 10px;
-                            `}
+            {showCompilationText ? (
```

Comment:
I think it is a good idea to split the compilation text here to a separate component.

Reply:
I agree. A formal component will be good. I have a PR in the works on the main repo to send the data as a formatted JSON object